PierreBrunetot/PulsechainConkyDashboard
# Optimizing Conky script parameters for better visibility

Summary of changes:

Updated setup.sh script: Added functionality for dynamic Conky configuration based on screen resolution.
New files added:
check_sync.sh: Script to check synchronization status.
validator_count.sh: Script to count validators.
validator_efficiency.py: Script to monitor validator efficiency.
images/demo_screenshot.png: Updated demo screenshot.
Conky configuration updates: Adjustments made to integrate new functionalities.
Why are these changes important?

These updates:

Enhance dynamic Conky configuration according to screen resolution, making the dashboard more adaptable to different screens.
Add additional monitoring scripts to track synchronization, validator count, and other important Pulsechain metrics.
Testing Instructions:

Install dependencies: Ensure that all necessary dependencies are installed (e.g., conky-all, python3, pip3, etc.).
Run the setup script: Execute setup.sh to update and configure your Conky setup.
Check the new scripts:
check_sync.sh: Verify that the synchronization status is displayed correctly.
validator_count.sh: Ensure that the validator count is accurately reported.
validator_efficiency.py: Confirm that the validator performance metrics are correctly reported.
Additional Notes:

Please refer to the updated documentation for specific instructions on configuring and using the new files and scripts.
Ensure that paths and addresses in the scripts are correctly set up for your specific environment.
